trade-finance-manager-ui/templates/case/parties/summary/bond.njk

```diff
@@ -5,9 +5,7 @@
 {% extends "index.njk" %}
 {% set edit = "/case/" + deal._id + "/parties/" + party %}
 
-{% block pageTitle %}
-    Check the unique reference number
-  {% endblock %}
+{% block pageTitle %}Check the unique reference number{% endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.